### DISCUSSION TITLE
Is there any way to disable the automatic `indent-style` detection?

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#10511] [Q&A]
Helix tries to infer indentation from the file, even if that overrides the language settings. While this makes sense from a convenience point, making it easier to adhere to the style of the current file; I find it mistakes C block comments for indented code, and sets the file's indentation to a single space.

Ideally there would be a way to turn this feature off globally, but I can't seem to find any.

**Comment by TornaxO7** [comment]
Well, maybe you could set `indent-style` to `1` which, I think, is basically the same.

**Reply by ghost** [reply]
I have the indent style set in `language.toml`, though, annoyingly helix overrides this on a per-file basis.

TBH, I have just learned to live with this minor annoyance.

**Reply by TornaxO7** [reply]
Hm... have you tried to set [`indent-heuristic`](https://docs.helix-editor.com/editor.html#editor-section) to `simple`?

**Reply by ghost** [reply]
It doesn't seem to help. Though I've just tried to recreate the issue and it seems to have been partially resolved; if the file already has correctly indented blocks, it indents properly.

**Reply by ghost** [reply]
To be honest, it doesn't annoy me that much anymore; I have just stopped using kernel-style block comments.

**Reply by vykt** [reply]
> Hm... have you tried to set [`indent-heuristic`](https://docs.helix-editor.com/editor.html#editor-section) to `simple`?

Can confirm this works with C block comments.
